packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewDetails.tsx

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
+
+import { Card, Column, Text, InfoItem } from '@trezor/components';
+import { GeneralPrecomposedTransactionFinal } from '@suite-common/wallet-types';
+import { spacings } from '@trezor/theme';
+
+import { Translation } from 'src/components/suite';
+
+const Pre = styled.pre`
+    text-align: left;
+    word-break: break-all;
+    white-space: pre-wrap;
+    font-family: monospace;
+`;
+
+export interface TransactionReviewDetailsProps {
+    tx: GeneralPrecomposedTransactionFinal;
+    txHash?: string;
+}
+
+const prettify = (json: Record<any, any>) => JSON.stringify(json, null, 2);
```

Comment:
Wow, I tried this one and it's pretty cool! I didn't know about this argument. TIL 😀